prisma/migrations/20230627061827_change_batch_payment_total_amount_type_to_int/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - You are about to alter the column `total_amount` on the `batch_payments` table. The data in that column could be lost. The data in that column will be cast from `DoublePrecision` to `Integer`.
+
+*/
+-- AlterTable
+ALTER TABLE "batch_payments" ALTER COLUMN "total_amount" SET DATA TYPE INTEGER;
```

Comment:
I think we can merge these two migrations into single one, just change the `total_amount` type in original migration.